{"name": "test_user_login_wrong", "status": "passed", "start": 1600240936555, "stop": 1600240936555, "uuid": "4794d9ed-e495-4a7c-a425-307f4c976495", "historyId": "8923e36b93bfcbf2b1b29bba9c76fef6", "testCaseId": "4b6bcd6bc9f4b93fe085c2939884ea45", "fullName": "testcase.test_login.TestLogin#test_user_login_wrong", "labels": [{"name": "severity", "value": "critical"}, {"name": "feature", "value": "测试登录模块"}, {"name": "story", "value": "测试登录密码错误"}, {"name": "parentSuite", "value": "testcase"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "TX5C3S1L7D31UYD"}, {"name": "thread", "value": "21264-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcase.test_login"}], "links": [{"type": "test_case", "url": "/basic/auth/login", "name": "/basic/auth/login"}]}